uint8 POS_ON=1
uint8 VEL_ON=2
uint8 ACC_ON=4
uint8 coordinate_frame
uint8 flag_mask
float64 x
float64 y
float64 z
float64 vel_x
float64 vel_y
float64 vel_z
float64 acc_x
float64 acc_y
float64 acc_z
---
bool success